vllm/sequence.py

```diff
@@ -130,6 +141,34 @@ def get_output_len(self) -> int:
     def get_token_ids(self) -> List[int]:
         return self.prompt_token_ids + self.output_token_ids
 
+    def advance_prefill_range(self, size: int) -> int:
```

Comment:
Change APIs to num_computed_tokens, record_num_computed_tokens, and reset_num_computed_tokens